frontend/app/views/fragments/promos/promoPrimary.scala.html

```diff
@@ -0,0 +1,28 @@
+@(
+    title: Html,
+    imageUrl: String,
+    stampImageOpt: Option[String] = None,
+    isLead: Boolean = false,
+    toneClass: Option[String] = None
+)(content: Html)
+
+@import views.support.Asset
+
+<div class="promo-primary@if(isLead){ promo-primary--lead}">
+    <div class="promo-primary__header @toneClass.mkString(" ")">
+        <h2 class="promo-primary__title h-promo">@title</h2>
+    </div>
+    <div class="promo-primary__media@if(stampImageOpt){ stamped-image}">
+        <div class="promo-primary__image">
+            <img src="@Asset.at(imageUrl)" alt="@title" class="responsive-img">
```

Comment:
As part of a follow up PR I think it would be worth getting these images into the Grid, so we can a) keep these assets out of the repo and b) more easily use Imager for responsive sizes.